### ISSUE TITLE
Constraint Violations not displayed to user

### ISSUE THREAD

**Issue by johnaohara** [Hyperfoil/Horreum#24]
If a user tries to create a new Schema that fails constraint violations, the user is shown a "Failed to save the Schema" message  

Any constraint violations are returned from the server in the HTTP 500 response e.g.

```
javax.validation.ConstraintViolationException: Validation failed for classes [io.hyperfoil.tools.repo.entity.json.Schema] during persist time for groups [javax.validation.groups.Default, ]
List of constraint violations:[
	ConstraintViolationImpl{interpolatedMessage='must not be null', propertyPath=access, rootBeanClass=class io.hyperfoil.tools.repo.entity.json.Schema, messageTemplate='{javax.validation.constraints.NotNull.message}'}
	ConstraintViolationImpl{interpolatedMessage='must not be null', propertyPath=owner, rootBeanClass=class io.hyperfoil.tools.repo.entity.json.Schema, messageTemplate='{javax.validation.constraints.NotNull.message}'}
]
```

These should be displayed in a meaningful way to the user.

Shouldn't the response be a HTTP 400 Bad Request, instead on Internal Server error?

**Comment by rvansa** [comment]
Good point. I guess you could get such exceptions from more points (where you access db) - some kind of interceptor?

**Comment by rvansa** [comment]
Implemented in e4661532d13239842775df954ce6ed337c58ccf0 and c1743890467d920ea73568ad61928db546f10ef9
